sherpa/astro/io/pyfits_backend.py

```diff
@@ -237,45 +285,76 @@ def read_table_blocks(arg, make_copy=False):
     return filename, cols, hdr
 
 
+# @DougBurke thinks that nobinary may just be a sign of a
```

Comment:
So, what to do here? My inclination would be to leave things as they are. Maybe add a TODO, maybe not.